### PR TITLE
Params fix

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java [hunk 1]
```diff
@@ -107,6 +107,9 @@ public VisualEditModePageObject open() {
   }
 
   public VisualEditModePageObject open(String articleName) {
+    String url = urlBuilder.appendQueryStringToURL(urlBuilder.getUrl() + URLsContent.WIKI_DIR + articleName,
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java]: MAJOR Remove this unused "url" local variable. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L156](https://github.com/Wikia/selenium-tests/blob/31278acbe27d4d831a325e089798cd40464be6e8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L156): Remove this unused private "verifyComponent" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L286](https://github.com/Wikia/selenium-tests/blob/31278acbe27d4d831a325e089798cd40464be6e8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L286): Reduce this lambda expression number of lines from 25 to at most 20. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [VisualEditModePageObject.java#L329](https://github.com/Wikia/selenium-tests/blob/31278acbe27d4d831a325e089798cd40464be6e8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java#L329): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/dataprovider/ArticleDataProvider.java [hunk 2]
```diff
@@ -17,7 +17,8 @@ private ArticleDataProvider() {
 
   @DataProvider
   public static final Object[][] articleTitles() {
-    return new Object[][]{{"QAarticleĄŻŁ"}, {"QAarticle國歷史"}, {"QAarticle/"},
+    return new Object[][]{
+//        {"QAarticleĄŻŁ"}, {"QAarticle國歷史"}, {"QAarticle/"},
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ArticleDataProvider.java]: Why would you not execute these 3 tests?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java (hunk 1 above)
Comment (ludwikkazmierczak) [reply src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java]: This one is not used anywhere, was there jusr for debugging

**Pull request merged**